### PR TITLE
Journeyman/restructure

### PR DESCRIPTION
**Pull request by journeyman-msft**
- [x] PR song: https://www.youtube.com/watch?v=tEoPy2a8kBI
- [x] cd to sample/financial_transcripts and run `python llm_app.py`
- [x] cd to sample/financial_transcipts and `streamlit run st_main.py --server.port 8000`
- [x] Observe how the skills come from a folder named `rag_skills`, which we will create a repo with the same content.

### CODE REVIEW

File: samples/financial_transcripts/llm_app.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 from chatBot import chatBot
 sys.path.append("../..")   ## add directory above
-from chatbotSkills import count_tokens
+from rag_skills.chatbotSkills import count_tokens
```
Comment (sdk-ai) [inline samples/financial_transcripts/llm_app.py:12]: `count_tokens` does not seem to be skill, in the strict sense of skill, or does it?

**Review by sdk-ai — COMMENTED** [review]
Tested all the steps in the review process. Restructure is as we discussed.

**Review by sdk-ai — APPROVED** [review]

**Comment by sdk-ai** [comment]
`utils` is fine.

**Comment by Enteli** [comment]
Sorry for being late.
- nice PR song!
- Structure of repo is as agreed
- I like the separation of skills and utils
- Next iteration (probably on rag-skills), i suggest we should consolidate/guide the user on what's in conversationalRetrievalwith...ipynb and demonstrateSkills.ipynb.  Why have 2 notebooks? Are all the skills used in the notebooks coming from chatbotSkills,py?

**Pull request merged**